src/Http/Controllers/Admin/MediaLibraryController.php

```diff
@@ -65,28 +96,28 @@ public function getIndexData($prependScope = [])
 
     protected function getRequestFilters()
     {
-        if (request()->has('search')) {
-            $requestFilters['search'] = request('search');
+        if ($this->request->has('search')) {
+            $requestFilters['search'] = $this->request->get('search');
         }
 
-        if (request()->has('tag')) {
-            $requestFilters['tag'] = request('tag');
+        if ($this->request->has('tag')) {
+            $requestFilters['tag'] = $this->request->get('tag');
         }
 
         return $requestFilters ?? [];
     }
 
     public function store($parentModuleId = null)
     {
-        $request = app(MediaRequest::class);
+        $request = $this->app->get(MediaRequest::class);
```

Comment:
This can be injected automatically as a method parameter.

Reply:
@yusufkandemir Actually not, since its parent class `ModuleController` has a method with the signature `public function store($parentModuleId = null)` so `MediaLibraryController`'s method can't set a different signature.

Reply:
Yes. Sorry about that, I reviewed this on Github instead of an IDE because I didn't have much time. It's hard to spot that kinda thing.